docs/reference/msbuild-targets.md

```diff
@@ -50,7 +50,9 @@ Note that the `Owners` and `Summary` properties from `.nuspec` are not supported
 | Description | Description | "Package Description" | |
 | Copyright | Copyright | empty | |
 | RequireLicenseAcceptance | PackageRequireLicenseAcceptance | false | |
-| LicenseUrl | PackageLicenseUrl | empty | |
+| license | PackageLicenseExpression | empty | Corresponds to <license type="expression"> |
+| license | PackageLicenseFile | empty | Corresponds to <license type="file">. You may need to explicitly pack the referenced license file. |
```

Comment:
check the preview link in Karan's comment. The license XML example gets treated as an unknown HTML tag, and is therefore not rendered. You probably need to HTML escape the < and >, or put it in a code block.